lib/plug/conn.ex

```diff
@@ -1205,6 +1219,13 @@ defmodule Plug.Conn do
       :nomatch -> value
     end
   end
+
+  defp request_url_port(:http, 80), do: ""
+  defp request_url_port(:https, 443), do: ""
+  defp request_url_port(_, port), do: [?:, to_string(port)]
```

Comment:
That's true. Haven't gotten around to changing it yet – want to leave it like this, then?

Personally I kind of like `:#{port}` for readability and brevity, though I understand it may be less efficient.